settings.py

```diff
@@ -189,3 +189,5 @@
 
 MAILROOM_URL = env('MAILROOM_URL', '')
 MAILROOM_AUTH_TOKEN = env('MAILROOM_AUTH_TOKEN', '')
+
+FLOW_SESSION_TRIM_DAYS = env('FLOW_SESSION_TRIM_DAYS', 7)
```

Comment:
Does this need to be `env.int`, or `int(...)`?

Reply:
Looks like its working fine as is, I set a env var for this test:
```
>>> from getenv import env
>>> FLOW_SESSION_TRIM_DAYS = env('FLOW_SESSION_TRIM_DAYS', 7)
>>> FLOW_SESSION_TRIM_DAYS==3
True
```

Reply:
Ah, maybe it tries to guess at the type if you don't specify? 👍